### ISSUE TITLE
forwardMessage

### ISSUE THREAD

**Issue by behnia1996** [justdmitry/NetTelegramBotApi#44]
Hello
How can I forward a message from a channel in my bot?
I find ID of channel and messageId but when use forwardMessage request in (await bot.makeRequestAsynd(myReq))  has error.

Text Error:
Exeption: Exception thrown:
'NetTelegramBotApi.BotRequestException' in mscorlib.dll
("Request failed (status code 400): Bad Request: chat not found")

**Comment by justdmitry** [comment]
According to [docs](https://core.telegram.org/bots/api#forwardmessage) you need `from_chat_id` as `@channelname`. Check that you don't forgot to add `@` before channelname.

I tried to forward message from channel to chat with bot and want it to be re-forwarded back to same chat with this code:

    if (update.Message.ForwardFromChat != null)
    {
        var req = new ForwardMessage(
            update.Message.Chat.Id, 
            "@" + update.Message.ForwardFromChat.Username, 
            update.Message.ForwardFromMessageId.Value);
        bot.MakeRequestAsync(req).Wait();
    }

And it works - bot re-forwarded message to me.

**Comment by justdmitry** [comment]
Also I tried with chat id instead of channel name:

    var req = new ForwardMessage(
        update.Message.Chat.Id, 
        update.Message.ForwardFromChat.Id, 
        update.Message.ForwardFromMessageId.Value);

Also works.

**Comment by behnia1996** [comment]
Thank you for your help. It was OK.
I have another question.

I can save editedMessage in bot, but I can't set editedMessage to null.
If editedMessage haven't null value, my server is busy because
update.EditedMessage haven't null value and in a lot of updates
editedMessage have'nt null value.

**Comment by justdmitry** [comment]
Please clarify what was wrong in your code and what "was Ok"? You forgot to add "@" before channelname?

About EditedMessages  - please create separate issue with more details, I don't understand what the problem is.

**Comment by behnia1996** [comment]
No, my problem in messageID section. But I did not know I should add
"@" before channelname. I use channelChatID in my bot.

my code for submit editedMessage is:

if (update.EditedMessage != null)
							{
								if (update.EditedMessage.Text != "1" &&
update.EditedMessage.Text != "2" && update.EditedMessage.Text != "3"
									&& update.EditedMessage.Text != "4" &&
update.EditedMessage.Text != "5" && update.EditedMessage.Text != "6")
								{
									dsLib.Point.updateAnatomicalAnswer
(update.EditedMessage.Text, update.EditedMessage.MessageId);
									var sendMessage = new SendMessage
(update.EditedMessage.Chat.Id, "Your Answer was
changed"+update.UpdateId.ToString()) { ReplyMarkup = days };
									await bot.MakeRequestAsync (sendMessage);
									update.EditedMessage.Text = "1";
									continue;
								}
								continue;
							}

But after change the message in my database this "If" also repeated
until bot have a new update. This repetition makes my server is busy
and message "Your answer was changed" is send to user.

I want to that message will only be sent once.
